Userland/Utilities/CMakeLists.txt

```diff
@@ -127,6 +127,7 @@ target_link_libraries(groups LibMain)
 target_link_libraries(gunzip LibCompress LibMain)
 target_link_libraries(gzip LibCompress LibMain)
 target_link_libraries(head LibMain)
+target_link_libraries(headless-browser LibWeb LibWebSocket LibGemini LibCore LibJS LibMarkdown LibGemini LibGfx LibTextCodec LibWasm LibXML LibHTTP LibMain)
```

Comment:
The rule of thumb is that you only need to add things here which you also include - in this case `LibCore LibGfx LibMain LibWeb` should be enough. Everything else is pulled in by LibWeb implicitly.

Reply:
Yes, that makes sense! I fixed it on a previous PR but not here. Done :)